packages/react-core/package.json

```diff
@@ -44,7 +44,6 @@
     "@types/jest": "^26.0.23",
     "@types/react": "^17.0.2",
     "@types/react-dom": "^17.0.13",
-    "@types/react-native": "^0.63.45",
```

Comment:
Removing `skipLibCheck` in `react-core` exposed the incompatibility issue with `@types/react-native` and `@types/node`. 

`@types/react-native` is unused in this package, and most likely will not be required in the future. If it is can turn back on `skipLibCheck` at that point